config/clusters/2i2c/mtu.values.yaml

```diff
@@ -0,0 +1,59 @@
+jupyterhub:
+  ingress:
+    hosts:
+      - mtu.2i2c.cloud
+    tls:
+      - secretName: https-auto-tls
+        hosts:
+          - mtu.2i2c.cloud
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "google"
+    homepage:
+      templateVars:
+        org:
+          name: Michigan Technological University
+          url: https://www.mtu.edu/
+          logo_url: https://www.mtu.edu/mtu_resources/images/download-central/logos/tech-vertical/full-color.png
+        designed_by:
+          name: 2i2c
+          url: https://2i2c.org
+        operated_by:
+          name: 2i2c
+          url: https://2i2c.org
+        funded_by:
+          name: 2i2c
+          url: https://2i2c.org
+  singleuser:
+    image:
+      name: quay.io/2i2c/cloudbank-data8-image
+      tag: "6286b77ae45c"
+  hub:
+    config:
+      Authenticator:
+        admin_users:
+          - dbkc
+          - lebrown
+      JupyterHub:
+        authenticator_class: cilogon
+      CILogonOAuthenticator:
+        oauth_callback_url: "https://mtu.2i2c.cloud/hub/oauth_callback"
+        shown_idps:
+          - http://google.com/accounts/o8/id
+          - https://sso.mtu.edu/idp/shibboleth
+        allowed_idps:
+          # Allow 2i2c staff to login with Google
+          http://google.com/accounts/o8/id:
+            username_derivation:
+              username_claim: "email"
+            allowed_domains:
+              - "2i2c.org"
+          # Allow MTU to login via Shibboleth
+          https://sso.mtu.edu/idp/shibboleth:
+            username_derivation:
+              username_claim: "email"
+              action: "strip_idp_domain"
+              domain: "mtu.edu"
```

Comment:
Was the intention here for the hub usernames to be stripped of the domains?
With this, instead of  `user@mtu.ed`u, the hub users will be just be `user`.

Also, since the rule for the google login doesn't have the stripping options, those ones will be left as emails.

Reply:
> Also, since the rule for the google login doesn't have the stripping options, those ones will be left as emails.

I think this has to be true for the `add_staff_user_ids_of_type` to work, otherwise it's comparing `sgibson` to `sgibson@2i2c.org` and not giving me admin rights. (I figured this out because I originally followed the example exactly with the 2i2c prefix.)

Reply:
I guess I find our CILogon docs confusing and more explanatory of what we _could_ do, not what we _should_ do.

Reply:
It also seems to not be working, so maybe I shouldn't be stripping the domains? https://github.com/2i2c-org/infrastructure/issues/2878#issuecomment-1677644549